plugins/modules/ipaservice.py

```diff
@@ -462,8 +618,16 @@ def main():
     state = ansible_module.params_get("state")
 
     # check parameters
+    if (names is None or len(names) < 1) and \
+       (services is None or len(services) < 1):
```

Comment:
This could be changed to `if not (names or services):` which, in my opinion reads better.